### ISSUE TITLE
PDFUA-178: Figure with no Alt text at all | FAIL

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#178]
Jira issue originally created by user Peter Spraul <p.spraul@axes4.com>:

This example demonstrates the failure condition that applies for an image that conveys information, but [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) is missing.

**Comment by zakkinsey** [comment]
Peter Spraul `<p.spraul@axes4.com>` assigned issue to self

**Comment by zakkinsey** [comment]
@MarkusErle changed assignee from Peter Spraul `<p.spraul@axes4.com>` to self

**Comment by zakkinsey** [comment]
Comment created by @MarkusErle:

Cleaned up with Roman's tool

**Comment by zakkinsey** [comment]
Comment created by @MarkusErle:

There is a direct relation of this example to a general failure condition: 
https://www.w3.org/WAI/WCAG21/Techniques/failures/F30

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #179: This issue relates to #179

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #221: This issue is cloned by #221

**Comment by zakkinsey** [comment]
Comment created by @BPeboeck:

I added Matterhorn Protocol 13-005 concerning procedure step 4.

**Comment by zakkinsey** [comment]
@willkil unassigned issue from @MarkusErle
